src/analysis/mean_inequalities.lean

```diff
@@ -30,19 +30,19 @@ and `n=3`.
 universes u v
 
 open real finset
-open_locale classical nnreal
+open_locale classical nnreal big_operators
 
 variables {ι : Type u} (s : finset ι)
 
 /-- Geometric mean is less than or equal to the arithmetic mean, weighted version
 for functions on `finset`s. -/
 theorem real.am_gm_weighted (w z : ι → ℝ)
   (hw : ∀ i ∈ s, 0 ≤ w i) (hw' : s.sum w = 1) (hz : ∀ i ∈ s, 0 ≤ z i) :
-  s.prod (λ i, (z i) ^ (w i)) ≤ s.sum (λ i, w i * z i) :=
+  (∏ i in s, (z i) ^ (w i)) ≤ s.sum (λ i, w i * z i) :=
```

Comment:
Might as well change `s.sum` as well. Also elsewhere.

Reply:
Thanks for doing this, btw :tada:

Reply:
I intend to focus on prod in this PR

Reply:
Aha, and then we simply
```
attribute [to_additive] #2989
```
once we're done :grinning: